package.json

```diff
@@ -30,7 +30,8 @@
   ],
   "dependencies": {
     "html-dom-parser": "0.1.2",
-    "react-dom-core": "0.0.2"
+    "react-dom-core": "0.0.2",
+    "style-to-object": "^0.2.0"
```

Comment:
Could you remove the `^`? I'm following `save-exact` since I don't have a `package-lock.json` or `yarn.lock` checked in.